support-frontend/assets/helpers/productCatalogue.ts

```diff
@@ -0,0 +1,276 @@
+export const productCatalog = window.guardian.productCatalog;
+
+export const productCatalogDescription = {
+	SupporterPlusWithGuardianWeekly: {
+		label: 'Digital + print',
+		benefitsSummary: [
+			'The rewards from ',
+			{ strong: true, copy: 'All-access digital' },
+		],
+		benefits: [
+			{
+				copy: 'Guardian Weekly print magazine delivered to your door every week  ',
+				tooltip: `Guardian Weekly is a beautifully concise magazine featuring a handpicked selection of in-depth articles, global news, long reads, opinion and more. Delivered to you every week, wherever you are in the world.`,
+			},
+		],
+	},
+	DigitalSubscription: {
+		label: 'The Guardian Digital Edition',
+		benefits: [
+			{
+				copy: 'The Editions app. Enjoy the Guardian and Observer newspaper, reimagined for mobile and tablet',
+			},
+			{ copy: 'Full access to our news app. Read our reporting on the go' },
+			{ copy: 'Ad-free reading. Avoid ads on all your devices' },
+			{
+				copy: 'Free 14 day trial. Enjoy a free trial of your subscription, before you pay',
+			},
+		],
+	},
+	NationalDelivery: {
+		label: 'National Delivery',
+	},
+	SupporterPlus: {
+		label: 'All-access digital',
+		benefits: [
+			{
+				copy: 'Unlimited access to the Guardian app',
+				tooltip: `Read beyond our 20 article-per-month limit, enjoy offline access and personalised recommendations, and access our full archive of journalism. Never miss a story with the Guardian News app – a beautiful, intuitive reading experience.`,
+			},
+			{ copy: 'Ad-free reading on all your devices' },
+			{
+				copy: 'Exclusive newsletter for supporters, sent every week from the Guardian newsroom',
+			},
+			{
+				copy: 'Far fewer asks for support',
+				tooltip: `You'll see far fewer financial support asks at the bottom of articles or in pop-up banners.`,
+			},
+		],
+	},
+	GuardianWeeklyRestOfWorld: {
+		label: 'The Guardian Weekly',
+	},
+	GuardianWeeklyDomestic: {
+		label: 'The Guardian Weekly',
+	},
+	SubscriptionCard: {
+		label: 'Newspaper subscription',
+	},
+	Contribution: {
+		label: 'Support',
+		benefits: [
+			{
+				copy: 'Exclusive newsletter for supporters, sent every week from the Guardian newsroom',
+			},
+		],
+	},
+	HomeDelivery: {
+		label: 'Home Delivery',
+	},
+};
+
+/** These `ratePlans` will eventually becomes part of the SupportPlus product in Zuora */
+export const supporterPlusWithGuardianWeekly = {
+	ratePlans: {
+		MonthlyWithGuardianWeekly: {
+			id: '',
+			pricing: {
+				USD: 43,
+				NZD: 67,
+				EUR: 36.5,
+				GBP: 25,
+				CAD: 46,
+				AUD: 57,
+			},
+			charges: {
+				Subscription: {
+					id: '',
+				},
+			},
+		},
+		AnnualWithGuardianWeekly: {
+			id: '',
+			pricing: {
+				USD: 480,
+				NZD: 760,
+				EUR: 413,
+				GBP: 275,
```

Comment:
Presume we are not aligning this with the regions like in the promoCods below because of Zuora? e.g. `GBP` could be replaced by `GBPCountries` (or vica-versa in Promos)?